### PR TITLE
fix: add Yaay

### LINKED ISSUE #32: 🚀 [Illustration] - Yaay
### Link of the Illustration

https://www.ls.graphics/illustrations/yaay

### Category of The Illustration

Paid

### Formats Available

Format
- Sketch
- Adobe Illustrator
- Adobe Xd
- Figma
- SVG
- Affinity

### Author

LS Graphics

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/Anmol-Baranwal/Awesome-Illustrations-4Projects/blob/main/CONTRIBUTING.md) of this project.

### PR DESCRIPTION
**Pull request by at-the-vr**
resolves #32 

## Name of Illustration
[Yaay](https://www.ls.graphics/products/yaay)

 

 

## Illustration Image

Workaholics_paid_illustrations

---

## Checklist

- [x] Name the PR fix: add `Yaay`.
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] You've linked the PR to the correct issue.
- [x] I have read the [Contribution Guidelines](https://github.com/Anmol-Baranwal/Awesome-Illustrations-4Projects/blob/main/CONTRIBUTING.md)

### CODE REVIEW

**Comment by Anmol-Baranwal** [comment]
Thanks for your contribution, we will soon check your PR @PiseAtharva :)

File: README.md [hunk 1]
```diff
@@ -357,6 +357,7 @@ Improve your projects using these awesome illustrations. See a sample <a href="h
...
 | <a href="https://growwwkit.com/illustrations/workaholics/"><img width='700' src='https://user-images.githubusercontent.com/74038190/194617817-1f9afa91-635b-4ee1-b4c6-dd87ff5647c3.png' alt="Workaholics_paid_illustrations" /></a> | [Workaholics](https://growwwkit.com/illustrations/workaholics/)  <br><br> 💰 Paid <br><br>Format <br> • AI<br>• SVG<br>• PNG<br>• PDF<br>• Illustrator<br>• Sketch<br>• Figma<br>• Corel Draw<br><br> ✌️Author <br> Growww  |
+| <a href="https://www.ls.graphics/products/yaay"><img width='700' src='https://user-images.githubusercontent.com/88548999/221354920-47a59d6c-f38b-41a0-ac85-dfbb26f75e0a.png' alt="Workaholics_paid_illustrations" /></a> | [Yaay](https://www.ls.graphics/products/yaay)  <br><br> 💰 Paid <br><br>Format <br> • Sketch<br>• Adobe Illustrator<br>• Adobe Xd<br>• Figma<br>• SVG<br>• Affinity<br><br> ✌️Author <br> LS Graphics  |
```
Comment (Anmol-Baranwal) [inline README.md]:
```suggestion
| <a href="https://www.ls.graphics/products/yaay"><img width='700' src='https://user-images.githubusercontent.com/88548999/221354920-47a59d6c-f38b-41a0-ac85-dfbb26f75e0a.png' alt="Yaay_paid_illustrations" /></a> | [Yaay](https://www.ls.graphics/products/yaay)  <br><br> 💰 Paid <br><br>Format <br> • Sketch<br>• Adobe Illustrator<br>• Adobe Xd<br>• Figma<br>• SVG<br>• Affinity<br><br> ✌️Author <br> LS Graphics  |
```

**Review by Anmol-Baranwal — CHANGES_REQUESTED** [review]
@PiseAtharva 

- Add this illustration in the Menu as well.

Do these changes in a single commit in this PR.

File: README.md (hunk 1 above)
Comment (at-the-vr) [reply README.md]: thanks for the suggestion

**Review by Anmol-Baranwal — APPROVED** [review]
Well done !!

**Comment by Anmol-Baranwal** [comment]
Appreciate your contribution :)

**Pull request merged**